### ISSUE TITLE
[/flow] English表示の時、769〜959pxの画面幅でテキストがはみ出す

### ISSUE THREAD

**Issue by oti** [tokyo-metropolitan-gov/covid19#1438]
## 改善詳細 / Details of Improvement

#1411 のサブイシューです。

英語版を表示して画面幅をPCレイアウトかつ960px未満にすると、テキストがボックスからはみ出したりしてレイアウトが崩れます

## スクリーンショット / Screenshot

画像が大きいのでリンクにしています。

[こちらは画面幅が 769px の時のキャプチャです。](https://user-images.githubusercontent.com/710216/76678400-e4adce00-661a-11ea-8316-4d6dc711357f.png)

「新型コロナ受診相談窓口」のセクションが顕著で、[950px あたりからテキストがはみ出し始めます。](https://user-images.githubusercontent.com/710216/76678409-05762380-661b-11ea-94e4-01b538529315.png)

## 期待する見せ方・挙動 / Expected behavior
- テキストがはみ出さないようにする

SPレイアウトのような作りだとテキストのはみ出しがないので、デザインを[SPレイアウト](https://user-images.githubusercontent.com/710216/76678697-3f482980-661d-11ea-861d-83dcb0768c74.png)に寄せられるなら寄せたいです。

デザイン変更する場合は #1440 も考慮したいただければ幸いです。

## 動作環境・ブラウザ / Environment
- macOS / Windows / Linux / iOS / Android
- Chrome / Safari / Firefox / Edge / Internet Explorer

**Comment by af12066** [comment]
- `word-break: break-word;`を入れる
- 下側のGridについて`grid-template-columns: repeat(2, 1fr);`を`grid-template-columns: repeat(2, 50%);`とする

とすると、ひとまずはみださなくなるが、文字の並びが不自然になる（とくに右下はテキストの位置自体を調整したほうがよさそう）
![dev-covid19-tokyo netlify com_en_flow_(iPad Pro)](https://user-images.githubusercontent.com/10768439/76681755-c7d4c300-6639-11ea-8956-df5ca2e3339a.png)

**Comment by munierujp** [comment]
flowページにおいてpc表示とsp表示のブレークポイントは現在`$medium`（`768px`）になっていますが、`$large`（`1170px`）にすれば解消されるでしょうか？

- [769px](https://user-images.githubusercontent.com/20086673/76683257-27d16680-6646-11ea-8650-e01491d8405e.png)
- [1171px](https://user-images.githubusercontent.com/20086673/76683251-11c3a600-6646-11ea-83da-5047560ae0a1.png)

**Comment by oti** [comment]
ありがとうございます。Flow画面についてはリデザインになるかどうかの議論がSlackで進められる予定でして、かといってそれを待っていてもずっと英語版が崩れたままになってしまうので、@af12066 さんの対応でいったんプルリクを作っていただいて良いと思います。

文字が隠れずに表示されていれば、読みづらくとも読むことはできるので…！

**Comment by af12066** [comment]
対応します
